### PR TITLE
Help article issue821 - disable platform auth

### PR DESCRIPTION
**Pull request by dewan-ahmed**
# What changed, and why it matters

Help migration article to resolve https://github.com/aiven/devportal/issues/821

I left out the CLI bits from [this article](https://help.aiven.io/en/articles/5086140-disabling-platform-authentication) and pointed the reader to the [dedicated CLI docs](https://docs.aiven.io/docs/tools/cli/account/account-authentication-method.html) instead.

### CODE REVIEW

**Comment by lornajane** [comment]
I'll try to review this soon, but can we figure out a more positive way to title this article, which is about securing an account rather than disabling anything? It's stop using your password now you have something better. But let's say that in a shorter way that is currently escaping me ...

**Comment by dewan-ahmed** [comment]
The securing part is on the SAML articles and this article only talks about how to disable platform authentication if you have enabled an alternative method of authentication. I thought of **Secure Aiven platform authentication** but then we're not talking about how to secure the platform authentication. I cannot think of an appropriate title that tells exactly what article is for but I'm happy to take suggestions.

**Comment by dewan-ahmed** [comment]
I also thought of "Swap password-based platform authentication" but we're not really swapping the two authentication here.

File: docs/platform/howto/disable-platform-authentication.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Disable platform authentication
+===============================
+
+If you have already :doc:`set up a SAML authentication <saml/saml-authentication>` method then you might want to disable the default password-based platform authentication method to prevent it being used to access projects within your account.
+
+Here are the steps to follow to disable platform authentication:
+
+1. Log in to the Aiven Console using your alternative authentication (for example, SAML) method. You must not be logged in with any open sessions using the password-based platform authentication.
```
Comment (ftisiot) [inline docs/platform/howto/disable-platform-authentication.rst:8]: Should the `You must not be logged in with any open sessions using the password-based platform authentication.` be a warning?

**Review by ftisiot — CHANGES_REQUESTED** [review]
Good content, a small suggestion for readibility

**Review by ftisiot — APPROVED** [review]
LGTM

**Comment by dewan-ahmed** [comment]
Thanks @ftisiot for adding the entries to the migration sheet 🙏🏽

**Comment by ftisiot** [comment]
:D

**Pull request merged**